docs/CSBR.md

```diff
@@ -487,39 +487,45 @@ Certificate issuance by the Root CA MUST require an individual authorized by the
-
-If the CA decides to conduct an investigation, it MUST inform the Application Software Supplier whether or not it will revoke the Certificate, within two (2) business days.
-
-If the CA decides that the revocation will have an unreasonable impact on its customer, then the CA MUST propose an alternative course of action to the Application Software Supplier based on its investigation.
-
-#### 4.9.1.2 Revocation Based on the Subscriber's Request
-
-The CA MUST revoke a Code Signing Certificate within one (1) business day if the Subscriber requests in writing that the CA revoke the Certificate or notifies the CA that the original certificate request was not authorized and does not retroactively grant authorization.
-
-#### 4.9.1.3 Revocation Based on Reported or Detected Compromise or Use in Malware
-
-For all incidents involving malware, CAs SHALL revoke the Code Signing Certificate in accordance with and within the following maximum timeframes. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these timeframes.
-
-1.  The CA MUST contact the software publisher within one (1) business day after the CA is made aware of the incident.
-2.  The CA MUST determine the volume of relying parties that are impacted (e.g., based on OCSP logs) within 72 hours after being made aware of the incident.
-3.  The CA MUST request the software publisher send an acknowledgement to the CA within 72 hours of receipt of the request.
-  a.  If the publisher responds within 72 hours, the CA and publisher MUST determine a "reasonable date" to revoke the certificate based on discussions with the CA.
-  b.  If CA does not receive a response, the CA must notify the publisher that the CA will revoke in 7 days if no further response is received.
-    i.  If the publisher responds within 7 days, the CA and the publisher will determine a "reasonable date" to revoke the certificate based on discussion with the CA.
-    ii.  If no response is received after 7 days, the CA must revoke the certificate except if the CA has documented proof (e.g., OCSP logs) that the revocation will cause significant impact to the general public.
-
-A CA revoking a Certificate because the Certificate was associated with signed Suspect Code or other fraudulent or illegal conduct SHOULD provide all relevant information and risk indicators to other CAs or industry groups. The CA SHOULD indicate whether its investigation found that the Suspect Code was a false positive or an inadvertent signing.
-
-#### 4.9.1.4 Revocation of a Subordinate CA Certificate
-
-As specified in BR Section 4.9.1.2.
+When revocation of a Subscriber Certificate is done due to a Key Compromise or use in Suspect Code the CA SHALL determine an appropriate value for the revocationDate based on its own investigation. The CA SHALL set a historic date as revocationDate if deemed appropriate.

```

Comment:
Something I think we missed in the latest revision of this change is the language that allows the exception cases where the Application Software Suppliers (e.g. Microsoft) will request that revocation be delayed due to immediate revocation having a large impact to the ecosystem. I suggest that we add this language here:

"The CA MAY delay revocation based on a request from Application Software Suppliers where immediate revocation has a potentially large negative impacts to the ecosystem."

I think this would cover this exception case to the rules in 4.9.1.1.

Reply:
IIRC, this was previously raised in a discussion regarding the ballot, but this provision/exception was dropped in the latest iteration of the language.

I think @ianjmcm's proposed language resolves this concern.

Reply:
Thank you @ianjmcm, I like that addition. I'll wait a few days to see if there are any other comments on that, before comitting